packages/dcos-log/extra/dcos-log-master.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=DC/OS Log service master: DC/OS logging service
+Description=DC/OS Log Master: exposes component, container, and task logs
```

Comment:
The log master doesn't expose the container/task logs from the agents? 
Do I need to hit the agents to get the agent logs or can i get them from the master?